<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSettings</class>
 <widget class="QWidget" name="WidgetSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>110</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>l5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditL6"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditL3"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>l3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>l6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>l4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditL4"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEditD"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>delta3</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEditTheta"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditL2"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEditD1"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>e</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="lineEditD5"/>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="lineEditD2"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>d</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>theta</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditL5"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>delta2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEditE"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditL1"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>l1</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>delta1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>l2</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEditPsi"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>psi</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>281</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSet">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonReset">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValid">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRobotPic">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>381</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
